tools/mmp/driver.cs

```diff
@@ -484,6 +487,8 @@ static void Pack (IList<string> unprocessed)
 
 			if (registrar == RegistrarMode.Default)
 				registrar = RegistrarMode.Dynamic;
+			if (is_extension)
+				registrar = RegistrarMode.Static;
 
```

Comment:
I think we can do something like the following:

```csharp

registrar = is_extension ? RegistrarMode.Static : RegistrarMode.Dynamic;
```

But I don't know if we really need to do the registrar == Default or not. If it is needed just leave the code as it is.